### DISCUSSION TITLE
unique plan

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#606] [Q&A]
We are seeing index (table-of-contents) files that have multiple reporting plan objects with the same values for all four fields that are used to describe they plan (plan_name, plan_id_type, plan_id, plan_market_type), but it seems like they are trying to describe different plans.  The entries with the same plan descriptions have different in_network_files and different allowed_amount_file values. So, while the plan key (the four plan parameters) are the same the files are different.

It seems like there is confusion about how a plan is supposed to be described with the four fields so that the key is unique.

I know there has been some discussion around what fields are used to describe a plan, however, the schema documentation isn't clear that in the table of contents file or in an in-network file, it is necessary to use a unique set of four parameters to describe the plan.

This seems to come up most often with group plans offered directly by an issuer.
In these cases, they are using their 5 digit HIOS code as the plan_id or their EIN, so each plan has the same plan_id and the same plan_id_type and the same plan_market_type of "group".  The issue is that the same plan_name is sometimes used rather than the actual name of the plan. An example of a plan_name is "Commercial Fully Funded". Since that isn't unique per plan, they end up with the TOC file containing several entries with the same plan key and these are fairly clearly meant to be different plans.

I am not sure why the plan_name field isn't being populated with a real name, but  documentation for the plan_name field may be part of the issue since it is fairly broad: "The plan name and name of plan sponsor and/or insurance company."  

Are others seeing this issue?  

We are seeing it enough that I suspect there is a misunderstanding of the need to make the plan "key" unique and the plan_name a real name.

**Comment by bryanbriegel** [comment]
We are seeing this, and I have lost track if I made a note about it here or not. To your point, we believe there needs to be additional guidance via the schema field Definition to provide explicit direction. As you wrote today "The plan name and name of plan sponsor and/or insurance company" allows differing interpretation of how to set the field value for plan_name.

More explicit guidance such as - 'For group health plans, the group health plan name and name of the plan sponsor associated with the EIN listed in plan_id. Optionally may additionally include the name of the insurance company.' 

For plan_id the Description could be updated to state: "For group health plans, the EIN of the plan sponsor. For health insurance companies, the 10-digit Health Insurance Oversight System (HIOS) identifier, or, if the 10-digit HIOS identifier is not available, the 5-digit HIOS issuer identifier, or if no HIOS issuer identifier is available, the Employer Identification Number (EIN) of the issuer." -- a revision in any fashion to provide clarity that MRFs for group health plans include the name of the group health plan, the plan sponsor and the EIN of the plan sponsor, and for plans offered by an issuer directly the HIOS plan ID, HIOS issuer ID or EIN of the issuers should be used, would help with standardization in file attribution.   

There are various interpretations of the schema/field requirements and this is yielding MRFs and rates that are difficult to attribute to a specific plan or issuer, as required by  the Departments.
